config/initializers/mime_types.rb

```diff
@@ -6,5 +6,8 @@
 # Mime::Type.register "text/richtext", :rtf
 # Mime::Type.register_alias "text/html", :iphone
 Mime::Type.register "image/svg+xml", :svg
+Mime::Type.register "application/smil+xml", :smil
+Mime::Type.register "application/x-mpegurl", :m3u8
```

Comment:
tested these on a branch app to ensure they don't interfere with existing routes like

https://github.com/antiwork/gumroad/blob/ff5ec881e28c0d59239dacb14f124271320a777a/config/routes.rb#L823-L824